CONTRIBUTING.md

```diff
@@ -19,8 +19,8 @@ Here's what you need to know about the tests:
   sure they are semantically equivalent.
 * Each test folder has a `jsfmt.spec.js` that runs the tests. Normally you can
   just put `run_spec(__dirname);` there. You can also pass options and
-  additional parsers, like this:
-  `run_spec(__dirname, { trailingComma: "es5" }, ["babylon"]);`
+  additional parsers, like this: `run_spec(__dirname, { trailingComma: "es5" },
+  ["babylon"]);`
```

Comment:
Should we disable the break in `inlineCode`? though it's breakable there.

Reply:
I personally would not break

Reply:
Yeah it's fine as is.

Reply:
Opened #3230